script.video.nfl.gamepass/resources/skins/Default/colors/defaults.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please drop this, I don't think scripts can define custom colors but rather use those provided by the skin...
It's one of those faults that have been in kodi since forever...along with fonts.